testbook/client.py

```diff
@@ -123,11 +130,20 @@ def inject(self, code, args=None, prerun=None):
         else:
             raise TypeError('can only inject function or code block as str')
 
+        inject_idx = len(self.cells)
+
+        if kwargs.get("after") is not None and kwargs.get("before") is not None:
```

Comment:
Why not just have `before=None, after=None` as named arguments?

Reply:
Sure, makes sense.